packages/manager/modules/pci/src/projects/project/data-processing/job-details/translations/Messages_fr_FR.json

```
@@ -0,0 +1,36 @@
+{
+  "data_processing_details_dashboard_label": "Tableau de bord",
+  "data_processing_details_logs_label": "Logs",
+  "data_processing_details_information_label": "Information",
+  "data_processing_details_information_status_label": "Statut",
+  "data_processing_details_information_region_label": "Région",
+  "data_processing_details_information_creation_date_label": "Créé",
+  "data_processing_details_information_elapsed_label": "Temps écoulé",
+  "data_processing_details_actions_show_ui_label": "Accéder à l'interface graphique du job",
+  "data_processing_details_actions_show_grafana_label": "Accéder au monitoring",
+  "data_processing_details_actions_browse_swift_label": "Parcourir les objets du conteneur",
+  "data_processing_details_actions_browse_swift_unavailable_label": "Le conteneur d'objets n'existe plus",
+  "data_processing_details_actions_relaunch_label": "Dupliquer et relancer le job",
+  "data_processing_details_actions_save_template_label": "Enregistrer le modèle de job",
+  "data_processing_details_actions_show_billing_label": "Accéder à la console de facturation",
+  "data_processing_details_actions_terminate_label": "Terminer le job",
+  "data_processing_details_actions_label": "Actions",
+  "data_processing_details_configuration_label": "Configuration",
+  "data_processing_details_configuration_cores_label": "Cœurs (driver/executors)",
+  "data_processing_details_configuration_memory_label": "Mémoire (driver/executors)",
+  "data_processing_details_configuration_executor_count_label": "# d'executors",
```

```suggestion
  "data_processing_details_configuration_executor_count_label": "Nombre d'executors",
```
Maybe a full sentence would be better